docs/content/tutorials/tutorial-batch.md

```diff
@@ -24,18 +24,99 @@ title: "Tutorial: Loading a file"
 
 # Tutorial: Loading a file
 
-## Getting started
-
 This tutorial demonstrates how to perform a batch file load, using Apache Druid (incubating)'s native batch ingestion.
 
 For this tutorial, we'll assume you've already downloaded Druid as described in 
 the [quickstart](index.html) using the `micro-quickstart` single-machine configuration and have it
 running on your local machine. You don't need to have loaded any data yet.
 
-## Preparing the data and the ingestion task spec
-
 A data load is initiated by submitting an *ingestion task* spec to the Druid Overlord. For this tutorial, we'll be loading the sample Wikipedia page edits data.
 
+An ingestion spec can be written by hand or you can use the "Data loader" that is built into the Druid console to help iteratively build one for you by sampling your data.
+The data loader currently only supports native batch ingestion (streaming support coming soon).
+
+We've included a sample of Wikipedia edits from September 12, 2015 to get you started.
+
+
+## Loading data with the data loader
+
+Navigate to [localhost:8888](http://localhost:8888) and click `Load data` in the console header, select `Local disk`.
```

Comment:
Navigate to [localhost:8888](http://localhost:8888) and click `Load data` in the console header. Select `Local disk`.